src/types/MultiversalAppBootstrapProps.ts

```diff
@@ -20,4 +20,5 @@ export declare type MultiversalAppBootstrapProps<PP extends MultiversalPageProps
   // See https://github.com/zeit/next.js/discussions/12558#discussioncomment-9177
   __N_SSG?: boolean;
   __N_SSR?: boolean;
```

Comment:
Why do we need to declare those `__N_SSG/SSR` types since they're privately used and might be even removed in the future?

Reply:
We don't "need" them, but I prefer to have those because it helps newcomers understanding what they are and how they should be used (or, not used)

If we didn't have them, people could mistakenly use them like traditional variables.